environment/lib/config.js

```diff
@@ -132,7 +132,11 @@ class RainbowConfig {
 // value is coming from.
 RainbowConfig.paramValues = {};
 
-RainbowConfig.optionalValues = ['bridge-src', 'core-src', 'nearup-src'];
+RainbowConfig.optionalValues = {
```

Comment:
I have remove this code in this PR https://github.com/near/rainbow-bridge/pull/108 since it is not necessary.